apm-lambda-extension/Makefile

```diff
@@ -24,10 +24,14 @@ all: build
 check-licenses:
 	go install github.com/elastic/go-licenser@v0.4.0
 	go run github.com/elastic/go-licenser@v0.4.0 -d .
+	go run github.com/elastic/go-licenser@v0.4.0 -d -ext .java .
+	go run github.com/elastic/go-licenser@v0.4.0 -d -ext .js .
 
 update-licenses:
 	go install github.com/elastic/go-licenser@v0.4.0
 	go run github.com/elastic/go-licenser@v0.4.0 .
+	go run github.com/elastic/go-licenser@v0.4.0 -ext .java .
+	go run github.com/elastic/go-licenser@v0.4.0 -ext .js .
```

Comment:
you wrote in the description:
> js: adding the license header cannot be automated as the formatting would be changed, but the check can be automated and is done

Isn't this line adding / updating the license headers for .js files?

Reply:
Removed that line now. It changes the formatting/intentation on the .js files. I actually updated the headers for js files manually and then run the control script via the `make check-licenses`.